admin_app/src/app/playground/page.tsx

```diff
@@ -202,6 +204,38 @@ const Page = () => {
       }
     }
   };
+
+  const sendResponseFeedback = (
+    message: ResponseMessage,
+    feedback_sentiment: FeedbackSentimentType,
+  ) => {
+    if (token) {
+      // Assuming message.json is a JSON string. Parse it if necessary.
+      const jsonResponse =
+        typeof message.json === "string"
+          ? JSON.parse(message.json)
+          : message.json;
+
+      const queryID = jsonResponse.query_id;
+      const feedbackSecretKey = jsonResponse.feedback_secret_key;
+
+      apiCalls
+        .postResponseFeedback(
+          queryID,
+          feedback_sentiment,
+          feedbackSecretKey,
+          token,
+        )
+        .then((response) => {
+          console.log("Feedback sent successfully: ", response.message);
+        })
```

Comment:
I was erroneously checking for `response.status == 200` when we're only getting json response. Stats check is handled by the function in api.ts